examples/cactus-example-discounted-asset-trade/balance-management.ts

```diff
@@ -41,7 +44,7 @@ export class BalanceManagement {
       // for Neo
       const contract = {}; // NOTE: Since contract does not need to be specified, specify an empty object.
       const method = { type: "web3Eth", command: "getBalance" };
-      const template = "default";
+      // const template = "default";
```

Comment:
hi
can you remove unused codes, including already commented out lines such as `// const method = "default";` below?
thanks

Reply:
Sorry you had to wait, but i somehow missed notification on github about your comment. Did what you asked for.